appveyor.yml

```diff
@@ -33,7 +33,7 @@ test_script:
   # Change to a non-source folder to make sure we run the tests on the
   # installed library.
   - "cd C:\\"
-  - "python -c \"import nose; nose.main()\" -v -s joblib"
+  - "py.test --disable-pytest-warnings --pyargs joblib"
```

Comment:
`--disable-pytest-warnings` flag is not available in older pytest versions. For python 2.7,, installing pytest without version specified installs `2.7.2`. As the pytest warnings are anyhow going to be cleared out later, I'm not adding this right now. We'll think of enforcing a version > 2.8 once Python 2.6 support is dropped...